src/app/util/ContentApp.cpp

```diff
@@ -46,20 +46,43 @@ namespace AppPlatform {
 EmberAfStatus ContentApp::HandleReadAttribute(ClusterId clusterId, chip::AttributeId attributeId, uint8_t * buffer,
                                               uint16_t maxReadLength)
 {
-    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d)",
+    ChipLogProgress(DeviceLayer, "Read Attribute for device %s cluster %d attribute=%d",
                     GetApplicationBasic()->GetApplicationName(), static_cast<uint16_t>(clusterId),
```

Comment:
Please fix the logging format strings to be correct instead of doing lossy casts on values (clusterId is a 32-bit value, as is attributeId).  This should be using the MEI log formatting macros.

Reply:
fixed with switch to ChipLogFormatMEI